src/integrations/database/definitions/constants.ts

```diff
@@ -0,0 +1,30 @@
+
+export const LogicalOperators =
+{
+    AND: '$and',
```

Comment:
These values are mongo specific. I expected the values "AND" and "OR", not he mongo variants